cmd/skaffold/app/cmd/flags.go

```diff
@@ -227,6 +227,15 @@ var FlagRegistry = []Flag{
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"dev", "run"},
 	},
+	{
+		Name:          "output",
```

Comment:
I'd name this as `--render-output` to clarify the scope in the name of the flag too

Reply:
Fixed